modules/nodes-pods-user-namespaces-configuring.adoc

```diff
@@ -0,0 +1,200 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-user-namespaces.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nodes-pods-user-namespaces-configuring_{context}"]
+= Configuring Linux user namespace support
+
+
+.Prerequisites
+
+* Enable the required Techology Preview features for your cluster by editing the `FeatureGate` CR named `cluster`:
++
+[source,terminal]
+----
+$ oc edit featuregate cluster
+----
++
+.Example `FeatureGate` CR
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: FeatureGate
+metadata:
+  name: cluster
+spec:
+  featureSet: TechPreviewNoUpgrade <1>
+----
+<1> Enables the required `UserNamespacesSupport` and `ProcMountType` features.
++
+[WARNING]
+====
+Enabling the `TechPreviewNoUpgrade` feature set on your cluster cannot be undone and prevents minor version updates. This feature set allows you to enable these Technology Preview features on test clusters, where you can fully test them. Do not enable this feature set on production clusters.
+====
++
+After you save the changes, new machine configs are created, the machine config pools are updated, and scheduling on each node is disabled while the change is being applied.
+
+* After the nodes return to a `READY` state, enable the crun container runtime to use with new containers on the worker nodes:
```

Comment:
I think it's better to describe why we must enable crun as default runtime. crun is currently the only released OCI runtime with support for user namespaces.

Reply:
I think it's not accurate to say : enable the crun container runtime to use with new containers on the worker nodes.
Indeed, for the existing containers on the workers, they will also use crun as runtime.